src/main/java/com/dongtronic/diabot/commands/nightscout/NightscoutSetDisplayCommand.kt

```diff
@@ -0,0 +1,66 @@
+package com.dongtronic.diabot.commands.nightscout
+
+import com.dongtronic.diabot.commands.DiabotCommand
+import com.dongtronic.diabot.data.NightscoutDAO
+import com.dongtronic.diabot.util.NicknameUtils
+import com.jagrosh.jdautilities.command.Command
+import com.jagrosh.jdautilities.command.CommandEvent
+import net.dv8tion.jda.core.entities.User
+import org.slf4j.LoggerFactory
+
+class NightscoutSetDisplayCommand(category: Command.Category, parent: Command?) : DiabotCommand(category, parent) {
+    companion object {
+        val validOptions = arrayOf("none", "title", "trend", "cob", "iob", "avatar")
+    }
+    private val logger = LoggerFactory.getLogger(NightscoutSetDisplayCommand::class.java)
+
+    init {
+        this.name = "display"
+        this.help = "Set display options for NS cards"
+        this.guildOnly = true
+        this.ownerCommand = false
+        this.aliases = arrayOf("setdisplay")
+        this.category = category
+        this.examples = arrayOf(this.parent!!.name + " display trend cob avatar", this.parent.name + " display reset (to reset)")
+    }
+
+    override fun execute(event: CommandEvent) {
+        val args = event.args.split("\\s+".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
+        val nickname = NicknameUtils.determineAuthorDisplayName(event)
+
+        when {
+            args.size == 1 && args[0] == "reset" -> {
+                resetNightscoutDisplay(event.author)
+                event.replySuccess("Reset Nightscout display options for $nickname")
+            }
+
+            args.isNotEmpty() -> {
+                // Prioritize `none` option over any others provided
+                val options = if(args.contains("none")) arrayOf("none") else args
+
+                // verify options and set
+                for(opt in options) {
```

Comment:
Add space between `for` and `(`

Reply:
This also counts for all the other `for`, `if`, and similar statements in your PR